src/pages/admin/reports/components/media-settings/MediaSettings.module.scss

```
@@ -0,0 +1,32 @@
+.error-button {
+    width: 200px;
+}
+
+.actions {
+    display: flex;
+    gap: 60px;
+    margin-top: 60px;
+    align-items: center;
+    padding-left: 385px;
+
+    Button {
```

Stylelint (17.4.0)

[error] Expected "Button" to be "button" (selector-type-case)

(selector-type-case)